tests/conftest.py

```diff
@@ -37,76 +50,44 @@ def make_tarfile(ids, tarfile_name="temp.tar", write_mode="w"):
     return tarfile_name
 
 
-small_content_1 = "small content 1"
-small_content_2 = "small content 2"
-small_content_3 = "small content 3"
-small_content_4 = "small content 4"
-big_content_1 = "".join(["1" for _ in range(HTTP_CHUNK_SIZE + 1)])
-big_content_2 = "".join(["2" for _ in range(HTTP_CHUNK_SIZE + 1)])
-big_content_3 = "".join(["3" for _ in range(HTTP_CHUNK_SIZE + 1)])
-big_content_4 = "".join(["4" for _ in range(HTTP_CHUNK_SIZE + 1)])
+big_content = (str(i) * (HTTP_CHUNK_SIZE + 1) for i in range(1, 5))
 
 uuids = {
-    "small": {
-        "contents": small_content_1,
-        "file_size": len(small_content_1),
-        "md5sum": md5(small_content_1),
-        "annotations": ["annotation 1"],
-        "related_files": ["related 1"],
-        "access": "controlled",
-    },
-    "small_ann": {
-        "contents": small_content_2,
-        "file_size": len(small_content_2),
-        "md5sum": md5(small_content_2),
-        "annotations": ["annotations.txt"],
-        "access": "open",
-    },
-    "small_rel": {
-        "contents": small_content_3,
-        "file_size": len(small_content_3),
-        "md5sum": md5(small_content_3),
-        "related_files": ["related 3"],
-        "access": "open",
-    },
+    "invalid": generate_metadata_dict(None, None, [], []),
+    "small": generate_metadata_dict(
+        "controlled", "small content 1", ["annotation 1"], ["related 1"],
+    ),
+    "small_ann": generate_metadata_dict(
+        "open", "small content 2", ["annotations.txt"], [],
+    ),
+    "small_rel": generate_metadata_dict("open", "small content 3", [], ["related 3"],),
+    "small_no_friends": generate_metadata_dict(
+        "controlled", "small content 4", [], [],
+    ),
+    "big": generate_metadata_dict(
+        "controlled", next(big_content), ["annotation 1"], ["related 1"],
```

Comment:
I think it'd make sense to make `big_content` a callable, instead of `generator`.

Reply:
I think you are right. Just to make sure I understand it correctly. 

Do you mean to use it like big_content(1)?